src/libs/auth.js

```diff
@@ -217,6 +220,9 @@ export const processUser = (user, isSignInEvent) => {
       cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(userId, cookiesAcceptedKey) : undefined,
       isTimeoutEnabled: isSignedIn ? state.isTimeoutEnabled : undefined,
       hasGcpBillingScopeThroughB2C: isSignedIn ? state.hasGcpBillingScopeThroughB2C : undefined,
+      // A user is an Azure preview user if they are a member of the Sam group _or_ they have the `azurePreviewUser` claim set from B2C.
```

Comment:
This logic is per slack thread: https://broadinstitute.slack.com/archives/C02GJ782BAA/p1672771575330129

I think this approach makes sense -- we can continue to manage the Sam group as we do today; but also allow tenants/users in B2C for cases where we can't check the Sam group (like `AzurePreview.js`).

Reply:
What does that mean for the backend code? We will have to add all users to both the Sam group and the B2C group in order for them to actually be able to create an Azure billing project.

Reply:
We will just remove the check in Rawls, @aherbst-broad made: https://broadworkbench.atlassian.net/browse/WOR-686

Today the check only gates creating an Azure billing profile. The thinking is the managed app offer is already allow-listed, and we check the publisher at create time, we don't need the additional group check in the backend.